RELEASE-NOTES.txt

```diff
@@ -9,6 +9,7 @@
 - [**] Products M4 features are now available to all. Products M4 features: add a simple/grouped/external product with actions to publish or save as draft. [https://github.com/woocommerce/woocommerce-ios/pull/3133]
 - [*] enhancement: Order details screen now shows variation attributes for WC version 4.7+. [https://github.com/woocommerce/woocommerce-ios/pull/3109]
 - [*] fix: Product detail screen now includes the number of ratings for that product. [https://github.com/woocommerce/woocommerce-ios/pull/3089]
+- [*] fix: Product subtitle now wraps correctly in order details. [https://github.com/woocommerce/woocommerce-ios/pull/3201]
```

Comment:
Note: [Already mentioned to the OP that this won't be picked up](https://github.com/woocommerce/woocommerce-ios/pull/3201#discussion_r529781665)